src/django/pfb_analysis/views.py

```diff
@@ -82,6 +84,17 @@ def perform_create(self, serializer):
                             name=slugify(serializer.validated_data['label']))
 
 
+class NeighborhoodGeoJsonViewSet(ReadOnlyModelViewSet):
+    """For retrieving neighborhood centroids as GeoJSON feature collection."""
+
+    queryset = Neighborhood.objects.all()
+    serializer_class = NeighborhoodGeoJsonSerializer
+    permission_classes = (IsAdminOrgAndAdminCreateEditOnly,)
+    pagination_class = None
+    filter_fields = ('organization', 'name', 'label', 'state_abbrev')
+    filter_backends = (DjangoFilterBackend, OrderingFilter, OrgAutoFilterBackend)
```

Comment:
This doesn't create a ton of duplication, but I think I would prefer a mixin to encapsulate the bits that are shared between this and the regular NeighborhoodViewSet.